README.md

```diff
@@ -313,6 +313,75 @@ You can configure the OpenTelemetry SDK for applications which can't currently b
 instrumentation.opentelemetry.io/inject-sdk: "true"
 ```
 
+### Target Allocator
+
+The OpenTelemetry Operator comes with an optional component, the Target Allocator (TA). When creating an OpenTelemetryCollector Custom Resource (CR) and setting the TA as enabled, the Operator will create a new deployment and service to serve specific `http_sd_config` directives for each Collector pod as part of that CR. It will also change the Prometheus receiver configuration in the CR, so that it uses the `http_sd_config` from the TA. The following example shows how to get started with the Target Allocator:
```

Comment:
It may be worth mentioning how this is a seperate service that could be deployed separately as well outside of the OpenTelemetry CRD. (It seems like docs about TA really should be more within the broader Otel docs rather than an Operator thing)

Reply:
It could possibly be deployed to a k8s cluster without the operator, but it depends on the k8s APIs and information provided by the operator in its configuration to identify collector instances amongst which it should allocate targets.  That's a fairly advanced use case I'm not sure we should try to document at this time as it is likely to add more confusion.

I agree that having documentation on the otel.io site would be good, but I'd suggest we start here and then work with the comms SIG to figure out what should live there and how to present it.